docs/src/main/asciidoc/spring-security.adoc

```diff
@@ -0,0 +1,252 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
+////
+= Quarkus - Quarkus Extension for Spring Security API
+
+include::./attributes.adoc[]
+
+While users are encouraged to use Java standard annotations for security authorizations, Quarkus provides a compatibility layer for Spring Security in the form of the `spring-security` extension.
```

Comment:
We should maybe have a link to the guide where we describe the standard annotations in the first part of the sentence.